qa/python_models/execute_grpc_error/config.pbtxt

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2024, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Is it possible to move the model to the `L0_backend_python/lifecycle` folder? I think it might be easier this way to track which test the models belong to.

Reply:
We don't have any models or model folder in
https://github.com/triton-inference-server/server/tree/main/qa/L0_backend_python/lifecycle
In test.sh we copy over from python_models and create the models folder with versions inside.
I have kept this new model in parallel to existing models used in `L0_backend_python/lifecycle`

Reply:
you can always create a `models` subfolder under `L0_*` tests

Reply:
thanks Olga, that's what I meant.

Reply:
In the existing test.sh we do
`rm -fr *.log ./models` before we start the test. 
https://github.com/triton-inference-server/server/blob/53200091b84f08a5e4921f5073137784570283e9/qa/L0_backend_python/lifecycle/test.sh#L37
For me to add models to `L0_backend_python/lifecycle` I will need to remove this.
We cleanup the models before every test, I will need to remove all the instances where we delete the models folder.
https://github.com/triton-inference-server/server/blob/53200091b84f08a5e4921f5073137784570283e9/qa/L0_backend_python/lifecycle/test.sh#L103
https://github.com/triton-inference-server/server/blob/53200091b84f08a5e4921f5073137784570283e9/qa/L0_backend_python/lifecycle/test.sh#L170
Not sure if I should change existing design might impact other tests too.

Reply:
name the folder differently and copy from it to `models` ?

Reply:
@tanmayv25 suggesnted I move all the changes from L0_lifecycle to a new L0_ dedicated for this feature going forward.
The reason I made changes here was, L0_lifecycle/models had models where we send errors pragmatically helped me reuse all the models.
Will resolve this comment along with the original comment by @tanmayv25 in a new PR after the cherrypick.
Keeping it unresolved for now will mark this resolved after the new PR